test/unit/client-side-encryption/state_machine.test.ts

```diff
@@ -190,27 +190,25 @@ describe('StateMachine', function () {
 
     context('when tls options are provided', function () {
       context('when the options are insecure', function () {
-        [
-          'tlsInsecure',
-          'tlsAllowInvalidCertificates',
-          'tlsAllowInvalidHostnames',
-          'tlsDisableOCSPEndpointCheck',
-          'tlsDisableCertificateRevocationCheck'
-        ].forEach(function (option) {
-          context(`when the option is ${option}`, function () {
-            const stateMachine = new StateMachine({
-              tlsOptions: { aws: { [option]: true } }
-            } as any);
-            const request = new MockRequest(Buffer.from('foobar'), 500);
-
-            it('rejects with the validation error', function (done) {
-              stateMachine.kmsRequest(request).catch(err => {
-                expect(err.message).to.equal(`Insecure TLS options prohibited for aws: ${option}`);
```

Comment:
is this error coming from mongo-client-encryption? if so, should we be removing these options from that code base as well? or should we leave them here and remove them whenever we are ready to remove them there?

Reply:
Oh good catch, that's coming from the state machine. I'll remove that.

Reply:
Those 3 options being tested here are still valid options for us and not related to OCSP. See this table for reference on how we use them: https://github.com/mongodb/node-mongodb-native/blob/main/src/mongo_client.ts#L1062